erc20-bridge-token/test/BridgeToken.js

```diff
@@ -298,4 +322,118 @@ describe('BridgeToken', () => {
       .be
       .revertedWith(`AccessControl: account ${user.address.toLowerCase()} is missing role ${ADMIN_ROLE}`);
   })
+
+  describe("Whitelist", function() {
+    let tokenInfo;
+    let testProofId = 1;
+    const recipient = "testrecipient.near";
+
+    beforeEach(async function() {
+      tokenInfo = await createEmptyToken(
+        nearTokenId,
+        BridgeTokenFactory,
+        BridgeTokenInstance
+      );
+      setMetadata(nearTokenId, tokenInfo.tokenProxyAddress);
+
+      const lockResultProof = require('./proof_template.json');
+      lockResultProof.outcome_proof.outcome.status.SuccessValue = serialize(
+        SCHEMA,
+        "LockResult",
+        {
+          prefix: RESULT_PREFIX_LOCK,
+          token: nearTokenId,
+          amount: 100,
+          recipient: ethers.utils.arrayify(adminAccount.address),
+        }
+      ).toString("base64");
+      const source = "123456789FGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz";
```

Comment:
Please name the variable accordingly (maybe `base58Alphabet`?)

Reply:
699afa2dcb032eb54349c0516950d007e028b2b1